File: backend/src/main/java/com/pgvaale/backend/entity/Feedback.java
package com.pgvaale.backend.entity;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Table(name = "feedback")
@Data
@NoArgsConstructor
@AllArgsConstructor
@Setter
@Getter
public class Feedback {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @ManyToOne
    @JoinColumn(name = "maid_id")
    private Maid maid;

    @ManyToOne
    @JoinColumn(name = "user_id")
    private User user;

    private String feedback;
    private Integer rating;
}

File: PGVaaleDotNetBackend/Entities/Feedback.cs
using System.ComponentModel.DataAnnotations;

namespace PGVaaleDotNetBackend.Entities
{
    public class Feedback
    {
        [Key]
        public long Id { get; set; }

        [Required]
        public long MaidId { get; set; }

        [Required]
        public long UserId { get; set; }

        public string FeedbackText { get; set; } = string.Empty;

        public int Rating { get; set; }

        // Navigation properties
        public Maid Maid { get; set; } = null!;
        public User User { get; set; } = null!;
    }
}
